Mathlib/FieldTheory/Normal.lean

```diff
@@ -283,6 +283,10 @@ theorem AlgEquiv.restrictNormal_trans [Normal F E] :
 def AlgEquiv.restrictNormalHom [Normal F E] : (K₁ ≃ₐ[F] K₁) →* E ≃ₐ[F] E :=
   MonoidHom.mk' (fun χ => χ.restrictNormal E) fun ω χ => χ.restrictNormal_trans ω E
 
+lemma AlgEquiv.restrictNormalHom_apply (L : IntermediateField F K₁) [Normal F L]
+    (σ : (K₁ ≃ₐ[F] K₁)) (x : L) : restrictNormalHom L σ x = σ x :=
+  AlgEquiv.restrictNormal_commutes σ L x
```

Comment:
Can you also add a version like `AlgEquiv.restrictNormal_commutes` that works in general and not just for intermediate fields?

Reply:
I think this lemma already exists, but I prefer to still have one for intermediate field because it is more frequently used.
<img width="617" alt="image" src="https://github.com/user-attachments/assets/66d40493-b292-4c73-86d7-30036c07dc8b">
This is only an unfold away from restrictNormalHom applying on an element.

Reply:
Fair enough. Although it does feel a bit weird to have `AlgEquiv.restrictNormal` have a general lemma but `AlgEquiv.restrictNormalHom` to have a lemma just for intermediate fields.